### ISSUE TITLE
Tabs/sections overlap the footer

### ISSUE THREAD

**Issue by ofek** [squidfunk/mkdocs-material#4063]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

![mk](https://user-images.githubusercontent.com/9677399/175850799-a9f7902e-9469-4169-8dba-fca0f354bca6.PNG)

### Expected behaviour

Scrollbar if too many

### Actual behaviour

Overlap

### Steps to reproduce

Many tabs

### Package versions

- Python: `3.8.1`
- MkDocs: `1.3.0`
- Material: `8.3.8`

### Configuration

```yaml
theme:
  name: material
  features:
    - navigation.sections
    - navigation.expand
    - navigation.tabs
    - navigation.tabs.sticky
    - navigation.instant

nav:
  - Plugins:
    - About: plugins/about.md
    - Builder:
      - Reference: plugins/builder/reference.md
      - Wheel: plugins/builder/wheel.md
      - Source distribution: plugins/builder/sdist.md
      - Custom: plugins/builder/custom.md
    - Build hook:
      - Reference: plugins/build-hook/reference.md
      - Version: plugins/build-hook/version.md
      - Custom: plugins/build-hook/custom.md
    - Metadata hook:
      - Reference: plugins/metadata-hook/reference.md
      - Custom: plugins/metadata-hook/custom.md
    - Environment:
      - Reference: plugins/environment/reference.md
      - Virtual: plugins/environment/virtual.md
    - Environment collector:
      - Reference: plugins/environment-collector/reference.md
      - Default: plugins/environment-collector/default.md
    - Publisher:
      - Reference: plugins/publisher/reference.md
      - PyPI: plugins/publisher/pypi.md
    - Version source:
      - Reference: plugins/version-source/reference.md
      - Regex: plugins/version-source/regex.md
      - Code: plugins/version-source/code.md
    - Version scheme:
      - Reference: plugins/version-scheme/reference.md
      - Standard: plugins/version-scheme/standard.md
    - Utilities: plugins/utilities.md
```

### System information

- Operating system: Windows
- Browser: Chrome

**Comment by squidfunk** [comment]
Thanks for reporting. Please provide a minimal reproducible example without overrides. I suspect that there are errors in the console, i.e., this is a side effect of some other error.

**Comment by ofek** [comment]
This branch https://github.com/pypa/hatch/tree/docs

**Comment by ofek** [comment]
Maybe a `mkdocs serve` or Windows issue, Linux deploy worked https://hatch.pypa.io/dev/plugins/about/

**Comment by squidfunk** [comment]
I'm closing this as non-reproducible for now. If you manage to create a minimal reproduction, please reopen.

**Comment by ofek** [comment]
Actually reproducible, go to https://hatch.pypa.io/dev/plugins/about/ then press `.` or click a page on the left. Basically the scrollbar goes away after the initial load

**Comment by squidfunk** [comment]
It's always a good idea to look into the inspector when that happens 😊

<img width="1435" alt="Bildschirmfoto 2022-07-02 um 06 42 35" src="https://user-images.githubusercontent.com/932156/176986811-36066024-2436-453d-94ee-eba1d124a933.png">

Maybe this helps to craft a _minimal_ reproducible example. I'd kindly ask you to narrow it down and attach it here as a zip, probably a quick fix then.

**Comment by squidfunk** [comment]
I think I know where this is coming from. Potential fix in 209856e72.

**Comment by ofek** [comment]
Oh interesting! The initial load works b/c that About page isn't affected. Thank you 🙇‍♂️
